crates/stack/Cargo.toml

```diff
@@ -10,7 +10,28 @@ description = "Embeddable networking stack"
 
 [dependencies]
 ya-relay-util = { workspace = true }
-ya-smoltcp = { version = "0.1" }
+smoltcp = { version = "0.10", features = [
+    "std",
+    "log",
+    "medium-ethernet",
+    "medium-ip",
+    "medium-ieee802154",
+    "phy-raw_socket",
+    "phy-tuntap_interface",
+    "proto-ipv4",
+    "proto-igmp",
+    "proto-dhcpv4",
+    "proto-ipv6",
+    "proto-ipv4-fragmentation",
+    "proto-sixlowpan-fragmentation",
```

Comment:
`fragmentation` features are new default features on `0.10`.
`0.8.1` had no `fragmentation` features at all, and I decided to include them.
I decided to not include new default features:
`socket-dns`
`socket-mdns`
`packetmeta-id`